tests/cxx/typedef_chain_in_template-d1.h

```diff
@@ -0,0 +1,22 @@
+//===--- typedef_chain_in_template-d1.h - test input file for iwyu --------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// This header mimics std::vector in libstdc++.
+
+#include "tests/cxx/typedef_chain_in_template-i1.h"
+
+template<typename T>
+class ContainerAsLibstdcpp {
+ public:
+  typedef T value_type;
+  typedef typename TypedefWrapper<T>::reference reference;
+  reference getContent() { return content_; }
```

Comment:
Here and throughout; you've started using camelCase rather than PascalCase for functions.